platforms/core-runtime/internal-instrumentation-api/src/main/java/org/gradle/internal/instrumentation/api/annotations/NotToBeReplacedByLazyProperty.java

```diff
@@ -24,6 +24,8 @@
 /**
  * Marks that a property will be kept as eager property also after migration.
  * This is a complementary annotation to {@link ToBeReplacedByLazyProperty}.
+ *
+ * This method is internal even though it's attached to a public API, so it should not be referenced in third-party code.
```

Comment:
❓ What is "this method" referring to here?

Reply:
Oh, it should be annotation... Good catch